README.md

```diff
@@ -71,16 +84,19 @@ NOTE: Using the [context](https://godoc.org/context) package, the user can pass
 to the underlying requests that the client makes. If you don't want to use this feature, then using `context.Background()`
 should be sufficient.
 
-For more sample code snippets, the [examples](https://github.com/infobip-community/infobip-api-go-sdk/tree/main/examples) directory
-contains tests intended to be used for live testing all endpoints. The prerequisite for running an individual test is changing
+
+### Examples
+
+The best way to learn how to use the library is to check the examples. For more sample code snippets, the [examples](https://github.com/infobip-community/infobip-api-go-sdk/tree/main/examples) directory
+contains tests intended to be used for live testing all endpoints of available channels. The prerequisite for running an individual test is changing
 the apiKey and baseURL variables, along with certain message fields, depending on the endpoint (e.g. From/To for WhatsApp).
 
 
-### Authentication ###
+## 🔐 Authentication
```

Comment:
I suggest that we move this part to the top, before installation. In my opinion, we follow devs workflow